<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChildForm</class>
 <widget class="QWidget" name="ChildForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>391</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label1">
       <property name="text">
        <string>Text1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label2">
       <property name="text">
        <string>Text2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label3">
       <property name="text">
        <string>Text3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label4">
       <property name="text">
        <string>Text4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label5">
       <property name="text">
        <string>Text5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label6">
       <property name="text">
        <string>Text6</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="QtWidgetsApplication1.qrc"/>
 </resources>
 <connections/>
</ui>
